Libs/Widgets/ctkMaterialPropertyPreviewLabel.cpp

```diff
@@ -238,6 +307,24 @@ void ctkMaterialPropertyPreviewLabel::draw(QImage& image)
   qreal specular = d->Specular;
   qreal specular_power = d->SpecularPower;
 
+  if (d->InterpolationPBR)
+    {
+    // Try to approximate PBR with Phong rendering.
+    // This is a very rough approximation, but instead of trying to make it
+    // more accurate, it would be better to accept an externally-rendered
+    // bitmap (as the appearance could be very complex, especially when
+    // textures and more sophisticated lighting is used).
+    diffuse = (1.0 - d->Metallic) * (0.75 + 0.25 * d->Roughness);
+    specular = d->Diffuse * (0.75 * (1.0 - d->Roughness));
+    specular_power = 20.0 + 30.0 * (1.0 - d->Roughness);
+    // Use ambient to make sure overall brightness (original diffuse lighting) is preserved
+    ambient = d->Diffuse - specular - diffuse;
+    if (ambient < 0)
+      {
+      ambient = 0;
+      }
+    }
+
```

Comment:
I don't think this preview does much good anyway - at least in Slicer you almost always have the live update of your model visible which is better than a sphere anyway.  I'm pretty sure this code is a holdover from an old dialog, maybe even ported from Tcl.